src/zap-borrow.ts

```diff
@@ -0,0 +1,31 @@
+import { Position, QuoteAPIResponseBody } from './types';
+import * as apisdk from '@protocolink/api';
+import { buildRouterTransactionRequest, quote } from './api';
+import * as common from '@protocolink/common';
+
+export interface ZapBorrowParams {
+  account?: string;
```

Comment:
```suggestion
  account: string;
```
Other files need to be corrected as well.

Reply:
fixed.